scripts/quick-helm-install.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+set -e
+
+echo "Adding the stackrox/helm-charts/opensource repository to Helm."
+
+helm repo add stackrox https://raw.githubusercontent.com/stackrox/helm-charts/main/opensource/
+
+echo "Generating stackrox-admin-password.txt"
+
+openssl rand -base64 20 | tr -d '/=+' > stackrox-admin-password.txt
+STACKROX_ADMIN_PASSWORD=`cat stackrox-admin-password.txt`
```

Comment:
Why do we need that file? For later checking the password?

I would rather propose the following:

```suggestion
echo "Generating stackrox-admin-password"
STACKROX_ADMIN_PASSWORD="$(openssl rand -base64 20 | tr -d '/=+')"
```

1. `.txt` is so windowsy, I think, we cad drop it as this is targeted towards Linux users
2. Prefer `$()` to the backticks
3. Skip the `echo "$STACKROX_ADMIN_PASSWORD"  > stackrox-admin-password` unless you want the password should survive longer than the life of that script
4. Maybe add an echo with suggestion that the user can 'cat stackrox-admin-password' anytime later

Reply:
Fair, I've added a line at the end of the script that prints out the password so the user can make sure to save it somewhere if he wants.